src/hooks/protections/receive_net_message.cpp

```diff
@@ -258,6 +258,22 @@ namespace big
 					}
 					break;
 				}
+				case rage::eNetMessage::MsgTransitionLaunchNotify:
+				{
+					if (frame->m_connection_identifier != gta_util::get_network()->m_transition_session.m_connection_identifier)
+					{
+						if (player)
```

Comment:
This check is useless as it checks if player is a `nullptr` before using the switch case.

Reply:
You can just block all remote `netMessages` by returning false in an else statement near the bottom.